Sources/Networking/Modifiers/Interceptors/LoggingInterceptor.swift

```diff
@@ -15,7 +15,7 @@ import Foundation
 // MARK: - Pretty logging modifier
 
 /// ``RequestInterceptor`` which logs requests & responses info into console in pretty way
-open class LoggingInterceptor: RequestInterceptor {
+public final class LoggingInterceptor: RequestInterceptor {
```

Comment:
Exactly, the inheritance is the problem unfortunately, also I was trying to use `@unchecked Sendable` only as the last resort. So I am not sure how we want to approach this.

Reply:
I'm not 100% sure either. My understanding is that if we are careful and we know, that our class is thread-safe, then we can use `@unchecked Sendable`, but with caution. Otherwise, I couldn't find any solutions about how to handle it and keep the inheritance 😢 . Did you see any? 

With those interceptors it should be straightforward right, as it doesn't have any state, or am I missing something?

Reply:
No I don't think that there is another way in case we want to keep the inheritance unfortunately 😞. But otherwise the interceptors should be thread safe. So are we ok with adding `@unchecked Sendable` then?